pkgs/development/libraries/kde-frameworks/kde-wrapper.nix

```diff
@@ -61,13 +61,9 @@ stdenv.mkDerivation {
         fi
     done
 
-    ln -s "$env/share" "$out"
```

Comment:
That's true. What we should probably do, then, is to switch from `mkDerivation` to `buildEnv` and set `pathsToLink = [ "/share"]`. We can set up the wrappers in a `postBuild` script. Does that sound good?